### DISCUSSION TITLE
goa_uniprot_all_noiea.gaf.gz missing GAF header?

### DISCUSSION THREAD

**Discussion by balvisio** [geneontology/helpdesk#361] [Q&A] [answered]
Hi all,

I have downloaded the filtered version of the GOA UniProt gene association file  Link: http://current.geneontology.org/annotations/goa_uniprot_all_noiea.gaf.gz

and noticed that it doesn't contain a GAF header. Is this an error or omission?

Thank you,

Bruno

**Comment by suzialeksander** [comment]
@kltm is this a pipeline thing?

**Comment by kltm** [comment, marked as answer]
Hm. I suspect that this is error by omission. Thank you for the heads up. I've added an issue here: https://github.com/geneontology/go-site/issues/1753 .
